test/test_experimental.py

```diff
@@ -1,28 +0,0 @@
-import pytest
-
-from torch_geometric import (
-    experimental_mode,
-    is_experimental_mode_enabled,
-    set_experimental_mode,
-)
-
-
-@pytest.mark.parametrize('options', [None, 'scatter_reduce'])
```

Comment:
I don't think we necessarily have to remove `experimental_mode` - there might be future features which might need this. For now, we can simply remove it from the documentation and remove the `scatter_reduce` option. WDYT?

Reply:
Yeah, good idea 😉. I was also considering this option. Code changed as suggested.